FluentSharp.CoreLib/O2_DotNetWrappers/Windows/Window.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+
+namespace FluentSharp.CoreLib.O2_DotNetWrappers.Windows
+{
+    public class Window
+    {
+        private delegate bool EnumWindowsProc(IntPtr hWnd, IntPtr lParam);
+        public const int WM_COMMAND = 0x0112;
+        public const int WM_CLOSE = 0xF060;
+
+        [DllImport("user32.dll")]
+        private static extern bool EnumWindows(EnumWindowsProc enumProc, IntPtr lParam);
+        [DllImport("user32.dll", CharSet = CharSet.Unicode)]
+        private static extern int GetWindowTextLength(IntPtr hWnd);
+        [DllImport("user32.dll", CharSet = CharSet.Unicode)]
+        private static extern int GetWindowText(IntPtr hWnd, StringBuilder strText, int maxCount);
```

Comment:
Can you make this methods public, since we are adding them, we might as well make them easily accessible :)